text/70_performance_testing.md

```diff
@@ -50,13 +50,13 @@ The easiest way to measure response time, of course, is to simply follow this al
 
 While this may be the easiest way to measure response time, it is far from the best.  There are a plethora of problems with this approach.  First, it is impossible to measure sub-second times; human response time is simply too variable and not fast enough.  You can't measure anything internal to the system if your only interface to the system is what you see.  It's very time-consuming, thus making it difficult to collect large datasets.  It's subject to human error (ever spend an entire day timing things?  At some point, you're bound to forget to click the "start" button, or accidentally click the "reset" button before you've written down the time).  It's an absolutely fantastic way to destroy tester morale (ever spend an entire day timing things?  At some point, you're bound to remember to go look for another job).  Because of all these issues (and more), performance testing is often done with the aid of tools.
 
-Although testing in general is using more automation, performance testing specifically tends to depend on automated tests.  There can be quite a bit of variation in performance indicators from run to run, due to other variables over which you may have little control.  Examples of these variables include other processes running on a server, how much physical RAM was already being used, garbage collection runs, and virtual machine startup times, among others.  Often the only way to get a truly valid result is running a performance test for numerous iterations and statistically analyzing it (obtaining the mean, median, and maximum response times, for example).  The only way to gain a reasonable number of samples in a reasonable amount of time is to automate the process.
+Although testing in general is becoming increasingly automated, performance testing in particular tends to depend on automated tests.  There can be quite a bit of variation in performance indicators from run to run, due to other variables over which you may have little control.  Examples of these variables include other processes running on a server, how much physical RAM was already being used, garbage collection runs, and virtual machine startup times, among others.  Often the only way to get a truly valid result is running a performance test for numerous iterations and statistically analyzing it (obtaining the mean, median, and maximum response times, for example).  The only way to gain a reasonable number of samples in a reasonable amount of time is to automate the process.
```

Comment:
More of a nitpick than the others. "Although testing in general is using more automation, performance testing specifically tends to depend on automated tests." was awkward to me. "Although testing in general is using" seemed to me like you were defining testing. Took me a second to see that you were conceding that _all_ testing is getting more automated, but performance testing is particularly so. Proposed change to make that more explicit.